readme.md

```diff
@@ -133,6 +133,23 @@ dependencies {
 }
 ```
 
+### Container Images
+
+You can specify container images that your product requires using the `artifacts` declaration on the `distribution`
+extension. For example, if a container could be pulled from `registry.example.io/foo/bar:v1.3.0`, you could add the
+following to the `distribution` extension:
+
+```gradle
+distribution {
+   artifact {
+        type = 'oci'
+        uri = 'registry.example.io/foo/bar:v1.3.0'
+    }
+}
+```
+
+The result will be embedded in the `deployment/manifest.yml` file.
```

Comment:
For manifest consumers, it might be worth giving a snippet of what this does to the `manifest.yml` file. Doesn't have to be the whole file, just the basic required ones and what it adds to the `extensions` object or whatever it's supposed to be.